scripts/build.sh

```diff
@@ -32,6 +32,7 @@ GIT_SHA=${GIT_SHA:-$(git rev-parse --short HEAD || echo "undefined")}
 GIT_SHA_FULL=${GIT_SHA_FULL:-$(git rev-parse HEAD || echo "undefined")}
 LDFLAGS="-X main.ExporterVersion=${VERSION} -X main.GitSHA=${GIT_SHA}"
 LDFLAGS="-X main.defaultGlusterd1Workdir=${GD1STATEDIR} -X main.defaultGlusterd2Workdir=${GD2STATEDIR}"
+LDFLAGS="-X main.defaultConfFile=${CONFFILE} -X main.defaultConfFile=${CONFFILE}"
```

Comment:
Double entry of main.defaultConfFile

Reply:
This variable is empty. `${CONFFILE} is not set.  Change Makefile as below

```
diff --git a/Makefile b/Makefile
index 735a70a..b873483 100644
--- a/Makefile
+++ b/Makefile
@@ -64,14 +64,14 @@ check-reqs:
 $(EXPORTER_BIN): $(EXPORTER_BUILD) gen-service
 $(EXPORTER_BUILD):
        FASTBUILD=$(FASTBUILD) BASE_PREFIX=$(BASE_PREFIX) GD1STATEDIR=$(GD1STATEDIR) \
+               CONFFILE=${SYSCONFDIR}/gluster-exporter/gluster-exporter.toml \
                GD2STATEDIR=$(GD2STATEDIR) ./scripts/build.sh $(EXPORTER)
        @echo
 
 install:
        install -D $(EXPORTER_BUILD) $(EXPORTER_INSTALL)
        install -D -m 0644 $(EXPORTER_SERVICE_BUILD) $(EXPORTER_SERVICE_INSTALL)
-       install -D -m 0600 ./extras/conf/global.conf.sample $(EXPORTER_CONF_INSTALL)/global.conf
-       install -D -m 0600 ./extras/conf/collectors.conf.sample $(EXPORTER_CONF_INSTALL)/collectors.conf
+       install -D -m 0600 ./extras/conf/gluster-exporter.toml.sample $(EXPORTER_CONF_INSTALL)/gluster-exporter.toml
        @echo
 
 vendor-update:
```

Reply:
Ack